tests/integration/conftest.py

```diff
@@ -417,14 +419,33 @@ def configure_local_internal_options(get_local_internal_options):
 
         yield
 
-        TimeMachine.time_rollback()
         conf.set_wazuh_local_internal_options(backup_options_lines)
 
         control_service('restart')
     else:
         yield
 
 
+@pytest.fixture(scope='function')
+def create_file_structure(get_files_list):
+    """ Creates the directory structure specified for a test case"""
```

Comment:
```suggestion
    """Creates the directory structure specified for a test case."""
```

The `Args` section is missing

Reply:
Done 5a5fabfccc0faab98a1ed77559881d0bebd9a227